cache/lru/lru.go

```diff
@@ -0,0 +1,44 @@
+package lru
+
+import (
+	lru "github.com/hashicorp/golang-lru"
+)
+
+// Cache encapsulates a thread-safe fixed size LRU cache
+// as defined in hashicorp/golang-lru.
+type Cache struct {
+	lru *lru.Cache
```

Comment:
The package is named lru, we import lru, we should avoid using lru also as a field name and it is actually  a cache.
```suggestion
	cache *lru.Cache
```